modules/openideal_statistics/openideal_statistics.tokens.inc

```diff
@@ -72,6 +72,7 @@ function openideal_statistics_tokens($type, $tokens, array $data, array $options
           $query = $entity_type_manager->getStorage('user')->getQuery();
           $result = $query
             ->condition('status', '1')
+            ->condition('uid', '1', '<>')
```

Comment:
1) We may have multiple administrator users.
2) Condition by user ID should not be used at all.